lib/integration/notifyConnector.ts

```diff
@@ -48,3 +49,50 @@ export const getNotifyInstance = () => {
 
   return notifyInstance;
 };
+
+export const sendEmail = async (email: string, personalisation: Record<string, string>) => {
+  try {
+    const templateId = process.env.TEMPLATE_ID;
+    if (!templateId && process.env.APP_ENV !== "test") {
+      throw new Error("No Notify template ID configured.");
+    }
+
+    const notify = getNotifyInstance();
+    await notify.sendEmail(templateId, email, { personalisation });
+  } catch (error) {
+    let errorMessage = "";
+
+    if (error instanceof AxiosError) {
+      if (error.response) {
+        /*
+         * The request was made and the server responded with a
+         * status code that falls out of the range of 2xx
+         */
+        const notifyErrors = Array.isArray(error.response.data.errors)
+          ? JSON.stringify(error.response.data.errors)
+          : error.response.data.errors;
+        errorMessage = `GC Notify errored with status code ${error.response.status} and returned the following detailed errors ${notifyErrors}.`;
+      } else if (error.request) {
+        /*
+         * The request was made but no response was received, `error.request`
+         * is an instance of XMLHttpRequest in the browser and an instance
+         * of http.ClientRequest in Node.js
+         */
+        errorMessage = `${error.request}.`;
+      }
+    } else if (error instanceof Error) {
+      errorMessage = `${(error as Error).message}.`;
+    }
+
+    logMessage.error(
```

Comment:
added the future alarm to https://docs.google.com/spreadsheets/d/1FXChe3eJq6BJgMBbF8Y_SXCnoOXoO3IiXMRWt7GoW0w/edit#gid=0